examples/src/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
+# Copyright 2020 Andy Maloney <asmaloney@gmail.com>
+
+project(Examples)
+
+set( APP_EXE ReadImages )
```

Comment:
Not sure using a variable for this is necessary.

If you're going to use one, please name it something descriptive like `IMAGES_EXAMPLE_TARGET`.